README.md

```diff
@@ -1152,6 +1154,27 @@ rm_downloaded: True # Extract script will remove downloaded zst after extraction
 rm_extracted: True # Preprocess script will remove extracted files after preproc.
 ```
 
+###### 5.1.2.4.4. LDDL
+<a id="markdown-51243-LDDL" name="51243-LDDL"></a>
+
+Language Datasets and Data Loaders (LDDL) is a utility library that minimizes the friction during dataset retrieval, preprocessing and loading for the language models.  LDDL provides dataset preprocesssing and dataloaders that allow for efficient training of Bert with dynamic sequence lengths in order to maximize training performance. To use LDDL with Nemo FW follow the directions provided [here] (https://github.com/NVIDIA/LDDL#bert) for preprocessing data with binning.
+
+With the data preprocessed in binned LDDL format the LDDL format can be used with the following changes to the YAML file:
+
+```yaml
+trainer:
+  data:
+    data_prefix: 
+      - /path/to/train/LDDL/Dataset
+      - /path/to/val/LDDL/Dataset
+      - /path/to/test/LDDL/Dataset
+    dataloader_type: LDDL
+
+```
+
+Note: Nemo FW currently only works with LDDL datasets that have been preprocessed with binning.
```

Comment:
Can you add instructions on how to preprocess the data for LDDL ? Also mention that you need to give all three (train/test and validation datasets) Coz it otherwise errors out.

Reply:
I can add more instructions.  but it does not error if you don't give all three https://github.com/NVIDIA/NeMo/pull/6761/files#diff-d496e911eb06fca3128061957d326a83ba0a603ca8482ad292aee3851f49bf61R535 there are checks